xo-server/package.json

```diff
@@ -22,13 +22,13 @@
   },
   "dependencies": {
     "load-json-file": "^2.0.0",
```

Comment:
Let's upgrade `load-json-file` and `@types/load-json-file` to their latest versions as well. Just to be consistent with upgrading dependencies :).